### PR TITLE
Add Lcov

### PR DESCRIPTION
**Pull request by lalten**
Add https://github.com/linux-test-project/lcov

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @mortenmj, modules you maintain (rules_perl) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (lcov, rules_cpan) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (lcov) have been updated in this PR. Please review the changes.

File: modules/lcov/metadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://github.com/linux-test-project/lcov",
+    "maintainers": [
+        {
+            "email": "bcr@laure.nz",
+            "github": "lalten",
+            "name": "Laurenz Altenmueller"
+        }
+    ],
+    "repository": [],
```
Comment (meteorcloudy) [inline modules/lcov/metadata.json:10]: Please add the source repo here.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/lcov/metadata.json (hunk 1 above)
Comment (lalten) [reply modules/lcov/metadata.json]: oops! fixed

**Comment by meteorcloudy** [comment]
Hmm, those files are essentially fetched by 
https://cs.opensource.google/bazel/bazel/+/master:tools/build_defs/repo/utils.bzl;l=106-114, which doesn't set the executable bit. I don't think we can even get this information at this point? /cc @fzakaria @Wyverald

**Comment by lalten** [comment]
Generating the .sh with `genrule(..., executable=True)` now.
I now wonder if it would also have worked to keep the overlay file but then patch the permission bits with a patch file?

**Comment by meteorcloudy** [comment]
> I now wonder if it would also have worked to keep the overlay file but then patch the permission bits with a patch file?

Yes, I think that's possible:
https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/repository/PatchUtil.java;l=459-468

**Review by meteorcloudy — APPROVED** [review]
Thanks!

**Pull request merged**